src/main/java/com/iexec/sms/tee/session/gramine/sps/SpsConfiguration.java

```diff
@@ -50,7 +50,7 @@ public String getWebUrl() {
     }
 
     public String getEnclaveUrl() {
```

Comment:
```suggestion
    public String getEnclaveHost() {
```
(https://developer.mozilla.org/en-US/docs/Web/API/URL/host)

Reply:
Right, thanks! Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/116/commits/8510a1eee205b04d1834a77a524cc4b475e28684.
(Also updated for CAS configuration to be consistent)

Reply:
Thanks for propagating update in CAS :)